erdjs/src/transaction.ts

```diff
@@ -10,9 +10,11 @@ import { TransactionPayload } from "./transactionPayload";
 import * as errors from "./errors";
 import { TypedEvent } from "./events";
 import { TransactionWatcher } from "./transactionWatcher";
-const createKeccakHash = require("keccak");
+import { ProtoSerializer } from "./proto";
+const createTransactionHasher = require("blake2b");
```

Comment:
there is also the version 2 tx, where we sign the tx hash, and there we use keccak
Is that also supported by erdjs? or will it be supported?

Reply:
For transactions V2, I've added JIRA task EN-8360 (we have to apply this for both `erdjs` and `erdpy`).